pusher/gcm.go

```diff
@@ -108,7 +107,7 @@ func (g *GCMPusher) configure(client interfaces.GCMClient, db interfaces.DB, sta
 			client,
 		)
 		if err == nil {
-			success++
+			g.MessageHandler[k] = handler
```

Comment:
gcm has no equivalent test for this change?

Reply:
No, because I needed the keys to have a successful case, and currently it only mocks the result. Do you think this is bad? I can try to find a key for testing, but since there isn't one already, I'm afraid this was not possible initially.

Reply:
I don't have a lot of context about Pusher. But I'd say if the effort is not high it might be worth it guaranteeing that they have equivalent implementations in regard to the fix you're applying.

Reply:
Cool, I'll see if it is easy to do that.